test/integration/prepare-ts-with-test-file/src/bar.test.ts

```diff
@@ -0,0 +1,5 @@
+describe("foo", () => {
+  it("bar", () => {
+    expect(true).toBe(true);
+  });
+});
```

Comment:
can you also run prettier format? I think the column can be erased, also will this file run in tests suite?